contracts/managers/AssetPairManager.sol

```diff
@@ -33,79 +33,103 @@ import { ITrigger } from "./triggers/ITrigger.sol";
  * @title AssetPairManager
  * @author Set Protocol
  *
- * Rebalancing Manager contract for implementing any trading pair strategy. Allocation determinations are made
- * base on output of Trigger contract. Max base asset allocation amount is passed in and used when bullish,
- * allocationPrecision - bullishBaseAssetAllocation used when bearish. Additionally, all allocations are priced
- * using the base contract's Allocator contract.
+ * Manager contract for implementing any trading pair and strategy. Allocation determinations are made
+ * based on output of Trigger contract. bullishBaseAssetAllocation amount is passed in and used when bullish,
+ * allocationDenominator - bullishBaseAssetAllocation used when bearish.
  */
 contract AssetPairManager {
     using SafeMath for uint256;
 
+    /* ============ Events ============ */
+
+    event InitialProposeCalled(
+        address indexed rebalancingSetToken
+    );
+
+    /* ============ Constants ============ */
+    uint256 constant private HUNDRED = 100;
+
     /* ============ State Variables ============ */
-    ICore public coreInstance;
-    IAllocator public allocatorInstance;
-    ITrigger public triggerInstance;
-    IAuctionPriceCurve public auctionLibraryInstance;
-    IRebalancingSetToken public rebalancingSetTokenInstance;
+    ICore public core;
+    IAllocator public allocator;
+    ITrigger public trigger;
+    IAuctionPriceCurve public auctionLibrary;
+    IRebalancingSetToken public rebalancingSetToken;
     uint256 public baseAssetAllocation;  // Percent of base asset currently allocated in strategy
-    uint256 public allocationPrecision;
+    uint256 public allocationDenominator;  //    
```

Comment:
To provide a suggestion, perhaps we can use `scalingFactor`, `mantissa` to describe this value